transforms/parse5Utils.js

```diff
@@ -1,51 +1,57 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
 import * as parse5Module from 'parse5'
 
 const parse5 = parse5Module.default ?? parse5Module
 
 export const walkParse5Ast = (node, cb, ctx) => {
-    if (cb) {
-        cb(node, ctx)
-    }
-
-    const children = node.tagName === 'template' ? node.content.childNodes : node.childNodes
-    // Not all parse5 AST nodes have childNodes
-    for (const child of children ?? []) {
-        walkParse5Ast(child, cb, ctx)
-    }
+  if (cb) {
+    cb(node, ctx)
+  }
+
+  const children = node.tagName === 'template' ? node.content.childNodes : node.childNodes
+  // Not all parse5 AST nodes have childNodes
+  for (const child of children ?? []) {
+    walkParse5Ast(child, cb, ctx)
+  }
```

Comment:
Why the whitespace changes? Maybe we should add prettier/eslint so we're not fighting about this. 😆

Reply:
Actually, it just occurred to me that I'm using `standard` for this project. You should be able to configure your IDE to apply `standard` styles automatically.

Reply:
I ran `npx standard --fix`, that should solve the spacing issue!

Reply:
On second look throguh, `npx standard --fix` might not have fixed the spacing issue.